src/test/java/com/artipie/rpm/CliArgumentsTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm;
+
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Tests for {@link CliArguments}.
+ *
+ * @since 0.9
+ */
+class CliArgumentsTest {
+
+    @Test
+    void canParseRepositoryArgument(@TempDir final Path temp) {
+        MatcherAssert.assertThat(
+            new CliArguments().parsed(
+                String.format(
+                    "%s",
+                    temp.getFileName()
+                )
+            ).repository(),
+            new IsEqual<>(temp.getFileName())
+        );
+    }
+
+    @Test
+    @Disabled
+    void canParseNamingPolicyArgument(@TempDir final Path temp) {
+        MatcherAssert.assertThat(
+            new CliArguments().parsed(
+                "-np sha1"
```

Comment:
@paulodamaso the correct short name for the argument is not `np` but is `n`: https://github.com/artipie/rpm-adapter/blob/master/src/main/java/com/artipie/rpm/CliArguments.java#L60. It should thus work.

Reply:
@victornoel even with `-n` this test doesn't work because parser considers the blank space after `-n` as part of the argument, and it can't find " sha1" value in enumeration. I think that it would be better to use `-n sha1` insteadh of `-nsha1` as argument, WDYT?

Reply:
@paulodamaso ok but this doesn't change the fact that `n` is the correct short argument, not `np`. so in that case maybe let's use the correct one and add a todo to solve that other problem (which won't be solvable with apache commons-cli IMHO). So either let's not use a short argument at all or leave it like that with both choices.

Reply:
@paulodamaso for the record, `argName` (here `np`) is just the name of the argument in the example showed on the output when using `--help`, see http://commons.apache.org/proper/commons-cli/properties.html